pkg/controller/garbagecollector/garbagecollector_test.go

```diff
@@ -213,12 +214,17 @@ func setupGC(t *testing.T, config *restclient.Config) garbageCollector {
 	if err != nil {
 		t.Fatal(err)
 	}
+	tweakableRM := meta.NewDefaultRESTMapper(nil)
+	rm := &testRESTMapper{meta.MultiRESTMapper{tweakableRM, testrestmapper.TestOnlyStaticRESTMapper(legacyscheme.Scheme)}}
```

Comment:
I don't understand why this changed.  If we are unable to use the previous logic, then we broke something.  This is a blocker for merge.

Reply:
reverted to original